### ISSUE TITLE
Odd large trees generating in villages

### ISSUE THREAD

**Issue by ShaneBeee** [SpigotMC/SpigotCraft#462]
- [ ] Bug exists on current Vanilla snapshot
- [x] Bug does not exist on current Vanilla snapshot
- [ ] Bug exists on current Spigot version (1.14.4)
- [x] Bug does not exist on current Spigot version (1.14.4)

It appears these odd, extremely large trees are generating in villages. This does not happen in vanilla, nor does this happen on the current Spigot 1.14.4

![2019-12-01_15 10 26](https://user-images.githubusercontent.com/20008482/69922027-2e4c2180-144d-11ea-9170-60893e6e2806.png)
![2019-12-01_15 10 31](https://user-images.githubusercontent.com/20008482/69922028-2e4c2180-144d-11ea-9c3e-a9f8820f90ed.png)
![2019-12-01_15 10 39](https://user-images.githubusercontent.com/20008482/69922029-2e4c2180-144d-11ea-87e4-106d0a24d7e8.png)
![2019-12-01_15 10 44](https://user-images.githubusercontent.com/20008482/69922030-2e4c2180-144d-11ea-81fc-99234386aad9.png)
![2019-12-01_15 10 51](https://user-images.githubusercontent.com/20008482/69922031-2e4c2180-144d-11ea-8acf-3a00482ea858.png)
![2019-12-01_15 11 02](https://user-images.githubusercontent.com/20008482/69922032-2ee4b800-144d-11ea-8f1e-b653a520ec2d.png)
![2019-12-01_15 11 08](https://user-images.githubusercontent.com/20008482/69922033-2ee4b800-144d-11ea-8c1a-e8f7a45175e6.png)

**Comment by ShaneBeee** [comment]
To add to this, these trees don't seem vanilla-like. I don't think I have ever seen an oak tree short and wide like this.

**Comment by md-5** [comment]
Coords?

**Comment by yannicklamprecht** [comment]
Additionally here are coord for another tree bug: -225 63 -741

**Comment by ShaneBeee** [comment]
-770, 71, 614 is the village im at right now.

**Comment by ShaneBeee** [comment]
Just to add to this, md_5 told me in game that trees changed in 1.15, this is all I can find regarding [**tree changes in 1.15**](https://minecraft.gamepedia.com/Java_Edition_1.15#World_generation)
```
World generation:
  Trees
    Birch trees generate in the dark forest biome once again.
```

**Comment by Puremin0rez** [comment]
The shapes of those trees resembles Swamp Trees the most I think.

Is this only happening at Villages?

**Comment by ShaneBeee** [comment]
> The shapes of those trees resembles Swamp Trees the most I think.
> 
> Is this only happening at Villages?

Its the only place I saw them. When flying around the village I didnt see any of these trees around the outskirts of the village.

**Comment by MrSpock78** [comment]
I found this:
https://bugs.mojang.com/browse/MC-166756

**Comment by Puremin0rez** [comment]
Yup, definitely a Vanilla issue. Interesting that exploring to the location versus teleporting directly there gives different results.

**Comment by ShaneBeee** [comment]
Ok good to know this is a vanilla issue. But that is VERY VERY odd that if you travel vs TP that it outcomes differently. Might be the oddest thing I have ever heard of. 

When I went to that village, I flew there (because the Minecraft:tp command I didnt have perm for, but at that point didnt realize I had perm for essentials TP command) hopefully Mojang fixes that, its a really odd bug. 


**CLOSED** due to being a vanilla issue.

**Comment by md-5** [comment]
I'll leave it open for tracking

**Comment by ShaneBeee** [comment]
> I'll leave it open for tracking

Ok cool cool. Just tested on Singleplayer/Vanilla and sure enough if I fly to the village (vs TP there), the weird trees are there. Super duper weird.

**Comment by Black-Hole** [comment]
https://bugs.mojang.com/browse/MC-166756 was fixed in 1.15-pre6
